modular_skyrat/modules/autoairlock_override/code/autoairlock.dm

```diff
@@ -0,0 +1,45 @@
+/// Check for the four states of open access. Emergency, Unrestricted, Security, and Engineering Override
+/obj/machinery/door/airlock/allowed(mob/user)
```

Comment:
Rename this file (and module) to `auto_airlock` please, it's not something that would go in one word.